cmds/logout.js

```diff
@@ -0,0 +1,23 @@
+const config = require('config');
+const configStore = require('../lib/configstore');
+const loginCmd = require('./login');
+
+exports.command = 'logout';
+exports.usage = 'logout';
+exports.description = 'Logs the currently authenticated user out of ReadMe.';
+exports.category = 'admin';
+exports.position = 2;
+
+exports.args = [];
+
+exports.run = async () => {
+  if (!configStore.has('email') || !configStore.has('project')) {
+    return Promise.reject(new Error(`Please login using \`${config.cli} ${loginCmd.command}\`.`));
```

Comment:
This error message doesn't make a whole lot of sense. Why would we prompt a user to login so that they can log out?

Reply:
Hm yeah, I guess we can just show the "you have logged out" message if they aren't logged in.